.github/CODEOWNERS

```diff
@@ -237,7 +237,7 @@
 /pkg/cmd/compile-build/      @cockroachdb/dev-inf
 /pkg/cmd/cr2pg/              @cockroachdb/sql-experience
 /pkg/cmd/dev/                @cockroachdb/dev-inf
-/pkg/cmd/docgen/             @cockroachdb/docs
```

Comment:
Hmm, I think you want to keep this line so that the other files in the `docgen` directory still have the correct owners.

Reply:
I'm not yet sure who owns that whole folder, but I know that I own `diagrams.go`, and docs is not the owner of the whole folder. I updated it to be unowned for the time being.

Reply:
CI might not let your PR through due to the unowned files. Let's wait and see if it passes.